### PR TITLE
feat: add swahili wordlist to bip39

### PR DESCRIPTION
**Pull request by Extheoisah**
There is currently no African language supported in the available set of wordlists used for generating mnemonics. With this PR, we are creating a wordlist for the swahili language. Swahili has an estimate of [200 million speakers](https://www.un.org/africarenewal/magazine/december-2021/swahili-gaining-popularity-globally) in the whole world.
We believe this will increase bitcoin adoption as some wallets ([Etta Wallet](https://github.com/crukundo/EttaWallet)) are looking forward to implement this wordlist.

The Swahili words were selected with the following rules:

1. Words can be uniquely determined typing the first 4 characters.
2. No accents or special characters.
3. No words with double spelling.
4. No offensive words.
5. No words already used in other language mnemonic sets.
6. No words with double spelling.
7. Words are 4-8 letters long.
8. No plural words.

The wordlist used for this implementation were derived from https://packetstormsecurity.com/files/32019/swahili.gz.html

All words have been carefully cross-checked by Swahili language experts.

Thanks to  @extheoisah @vladimirfomene @brandonodiwuor (faith faithmakena32@gmail.com) ([Paul Warambo](https://www.proz.com/profile/1695794)) for their contribution to this project.

### CODE REVIEW

**Comment by dunxen** [comment]
Strong concept ACK

File: bip-0039/bip-0039-wordlists.md [hunk 1]
```diff
@@ -115,3 +116,20 @@ Credits: @alegotardo @bitmover-studio @brenorb @kuthullu @ninjastic @sabotag3x @
...
+Credits: @extheoisah @vladimirfomene @brandonodiwuor ([faith](faithmakena32@gmail.com)) ([Paul Warambo](https://www.proz.com/profile/1695794))
+
+The Swahili words were selected with the following rules:
+
+1. Words can be uniquely determined typing the first 4 characters.
+2. No accents or special characters.
+3. No words with double spelling.
+4. No offensive words.
+5. No words already used in other language mnemonic sets.
+6. No words with double spelling.
```
Comment (okjodom) [inline bip-0039/bip-0039-wordlists.md:131]: Dup of comment line # 3 `No words with double spelling.`

**Review by axljoker03 — APPROVED** [review]

**Comment by okjodom** [comment]
Given the expected permanence of this wordlist once accepted and merged, I proposed to work with @vladimirfomene to devise a reproducibility test, starting with a differentiated corpus

**Comment by vladimirfomene** [comment]
Based on this comment: https://github.com/bitcoin/bips/pull/1359#issuecomment-1262880949, this PR will have to be moved to a different repository.

**Comment by dunxen** [comment]
> Based on this comment: https://github.com/bitcoin/bips/pull/1359#issuecomment-1262880949, this PR will have to be moved to a different repository.

Alright, makes sense to close and move then 👍

**Comment by murchandamus** [comment]
It appears that [the author(s) of BIP 39 have decided not to accept any further word lists into BIP 39 itself, and encourage adding new ones to the WLIPs repo](https://github.com/bitcoin/bips/pull/1129#issuecomment-873258377) here: https://github.com/p2w34/wlips

If BIP39 no longer accepts new wordlists, and such pull requests should instead be directed at the WLIPs repository, it seems to me that this PR can be closed.

I propose to close this PR after 2024-05-10 unless further activity indicates otherwise.

**Review by Dorex45 — COMMENTED** [review]
PR ACK,

**Pull request closed without merging**